### PR TITLE
feat(file-history): Added `--before` and `--after`

### PR DESCRIPTION
**Pull request by emmanueltouzery**
support for git log flags.

### CODE REVIEW

**Review by sindrets — CHANGES_REQUESTED** [review]
Thanks for the PR! This looks pretty good. Please just amend [the type definition for `GitLogOptions`](https://github.com/emmanueltouzery/diffview.nvim/blob/df507ddea4f064f1a750f429230ebe373ba196fe/lua/diffview/config.lua#L254) and the docs for `:h :DiffviewFileHistory`.

**Comment by sindrets** [comment]
In the docs it could also be worth mentioning that the flags accept dates in natural language as well (like i.e. `"2 weeks 6 hours 12 minutes ago"`).

File: lua/diffview/vcs/adapters/git/init.lua [hunk 1]
```diff
@@ -1787,6 +1791,12 @@ GitAdapter.flags = {
     FlagOption("=S", "-S", "Search occurrences", {
       prompt_label = "(Extended regular expression)"
     }),
+    FlagOption("=A", "--after==", "List only commits after a certain date", {
```
Comment (sindrets) [inline lua/diffview/vcs/adapters/git/init.lua:1794]: Should only be one equal sign: `--after=`

**Comment by emmanueltouzery** [comment]
applied the requested changes :+1:

**Review by sindrets — APPROVED** [review]
LGTM, thank you! I made a small change adding Git's synonyms for these flags `--since` and `--until`.

**Comment by emmanueltouzery** [comment]
:+1: on the since & until.

**Pull request merged**